nixos/modules/services/web-apps/shiori.nix

```diff
@@ -39,9 +39,57 @@ in {
 
       serviceConfig = {
         ExecStart = "${package}/bin/shiori serve --address '${address}' --port '${toString port}'";
+
         DynamicUser = true;
-        Environment = "SHIORI_DIR=/var/lib/shiori";
+        Environment = [ "SHIORI_DIR=/var/lib/shiori" ];
```

Comment:
I usually prefer to use attribute sets like so:
```
systemd.services.shiori.environment.SHIORI_DIR = "/var/lib/shiori";
```

Of course either way works.

@bjornfor this PR looks good and has your approval, any reason we should hold back on merging?

Reply:
I don't mind changing it. Out of curiosity though, is there an advantage to doing it this way, apart from allowing other modules to easily access it?

Reply:
Not much more of an advantage in this case, aside from what you said. In general using attribute sets over lists allows greater ability to override of course, so I favour that.

Like I said though, either way works.